### DISCUSSION TITLE
any way to skip mandatory updates?

### DISCUSSION THREAD

**Discussion by brobcoolguy** [3kh0/ext-remover#1016] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
previously my chromebook was on v119 and i used caub (.onc method) but then the admins pushed a mandatory immediate update for everyone to v121 and im pretty sure skiovox is patched in that ver. i cant sign in or go to apps without updating, is there any way to skip these updates or am i cooked??

**Comment by Blobby-Boi** [comment, marked as answer]
Use caudns

**Reply by v4pl** [reply]
Ya caubdns
